src/main/java/hybrid/it/internship/library/entity/Book.java

```diff
@@ -0,0 +1,39 @@
+package hybrid.it.internship.library.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Getter @Setter
+    @Id
```

Comment:
Since all entities will have the same identifier column you can introduce Abstract class containing ID field inherited by all entities